session.go

```diff
@@ -43,6 +44,10 @@ type Session interface {
 	// used it will return nil.
 	PublicKey() PublicKey
 
+	Context() context.Context
```

Comment:
Would it be worthwhile ensuring there are methods for everything exported by the ssh.Context? Especially if we're returning this as a context.Context here.